### PR TITLE
doc: 17.2.8 Release Notes with CERN el8 builds

### PR DESCRIPTION
**Pull request by ebocchi**
## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "quincy"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

- When filling out the below checklist, you may click boxes directly in the GitHub web UI.  When entering or editing the entire PR message in the GitHub web UI editor, you may also select a checklist item by adding an `x` between the brackets: `[x]`.  Spaces and capitalization matter when checking off items this way.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [x] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [x] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
- `jenkins test rook e2e`
</details>

### CODE REVIEW

File: doc/releases/quincy.rst [hunk 1]
```diff
@@ -1457,6 +1457,20 @@ Changelog
 
 - win32\_deps\_build<span></span>.sh: change Boost URL ([pr#55085](https://github.com/ceph/ceph/pull/55085), Lucian Petrut)
 
+
+el8 builds                                                                       
+----------
+
+RPM packages for el8 are provided by CERN as a community member. These packages
```
Comment (anthonyeleven) [inline doc/releases/quincy.rst:1464]: EL8 (RHEL, CentOS, Rocky, Alma, etc)

File: doc/releases/quincy.rst [hunk 2]
```diff
@@ -1457,6 +1457,20 @@ Changelog
...
+come with no warranty and have not been tested. The software in them has been
+tested by Ceph according to [platforms](https://docs.ceph.com/en/latest/start/os-recommendations/#platforms).
+
+The repository for el8 builds is hosted by CERN on [Linux @ CERN](https://linuxsoft.cern.ch/repos/ceph-ext-quincy8el-stable/).
```
Comment (anthonyeleven) [inline doc/releases/quincy.rst:1468]: EL8
s/on/at/

**Review by anthonyeleven — COMMENTED** [review]
A few suggestions

File: doc/releases/quincy.rst (hunk 1 above)
Comment (ebocchi) [reply doc/releases/quincy.rst]: Done!

File: doc/releases/quincy.rst (hunk 2 above)
Comment (ebocchi) [reply doc/releases/quincy.rst]: Done, many thanks!

**Comment by anthonyeleven** [comment]
Be sure to tick the checklist items

**Comment by anthonyeleven** [comment]
jenkins retest this please

File: doc/releases/quincy.rst [hunk 3]
```diff
@@ -1457,6 +1457,21 @@ Changelog
 
 - win32\_deps\_build<span></span>.sh: change Boost URL ([pr#55085](https://github.com/ceph/ceph/pull/55085), Lucian Petrut)
 
+
+el8 builds                                                                       
+----------
+
+RPM packages for EL8 (RHEL, CentOS, Rocky, Alma, etc.) are provided by CERN as a
```
Comment (alexiri) [inline doc/releases/quincy.rst:1464]:
```suggestion
RPM packages for EL8 (RHEL, CentOS, Rocky, AlmaLinux, etc.) are provided by CERN as a
```

**Comment by zdover23** [comment]
I made an error here. I'll fix it.

File: doc/releases/quincy.rst (hunk 3 above)
Comment (zdover23) [reply doc/releases/quincy.rst]: This is repaired.

**Comment by zdover23** [comment]
@yuriw, As far as I'm concerned, this is ready to go. I don't want to jump the gun on this, so I'll wait for your okay before pressing the `Approve and run` button.

**Comment by zdover23** [comment]
When these commits are squashed and Enrico's commit overwrites the line in `quincy.rst` that in Yuri and Laura's commit contains a literal string that has a start but not an end, the make check should work properly.

I suppose it is possible that I could alter the commit in the `wip-yuriw-release-17.2.8-main` branch to resolve this conflict, but it would be the first time that I had altered both the origin branch and the target branch.

**Comment by zdover23** [comment]
jenkins test make check

**Pull request closed without merging**